lib/pure/collections/deques.nim

```diff
@@ -109,9 +109,13 @@ proc toDeque*[T](x: seq[T]): Deque[T] =
   result.head = 0
   result.count = x.len
   result.tail = x.len
-  result.data = x
+  if x.len.isPowerOfTwo:
+    result.data = x
   if not x.len.isPowerOfTwo:
```

Comment:
replace
`if not x.len.isPowerOfTwo:`
by
`else:` 

(like i wrote in https://github.com/nim-lang/Nim/pull/13620#discussion_r391967294 ...)

Reply:
Woops, that's what I get for rushing..